invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/utlis/CopyButton.js

```diff
@@ -0,0 +1,94 @@
+// This file is part of InvenioRDM
```

Comment:
comment not on the content, but on the package itself. There was already a utils.js file, but this is creating a folder.... shall we try to put all these "generic things in more meaningfull named packages/folder? e.g. we have `EditButton` in the langind page, but this goes to utils... Shall we create a `buttons` folder... random thought on code organization hehe

Reply:
Yes please! No utils packages/files whenever possible!
The CopyBtn cmp might go in the future in some common lib to be reusable...

Reply:
Yes, I will change the directory name (which  was actually misspelled :upside_down_face:)  to `buttons`